vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -136,7 +137,10 @@ void FlyoutShadowNode::createView()
   Super::createView();
 
   m_flyout = winrt::Flyout();
-  m_showOptions = winrt::FlyoutShowOptions();
+  m_isFlyoutShowOptionsSupported = !!(m_flyout.try_as<winrt::IFlyoutBase5>());
```

Comment:
It's possible to make m_isFlyoutShowOptionsSupported as local variable like isFlyoutShowOptionsSupported

Reply:
If I do that, I'd need to refactor the code to make m_showOptions local also. I'd rather not make that changes without finding out from Ashita if she had more plans to set more of these options.